iohk-monitoring/test/Cardano/BM/Test/Configuration.lhs

```diff
@@ -270,12 +270,13 @@ unitConfigurationParsedRepresentation = do
             , "  mapMonitors:"
             , "    chain.creation.block:"
             , "      actions:"
-            , "      - AlterMinSeverity \"chain.creation\" Debug"
+            , "      - (CreateMessage Warning \"chain.creation\")"
+            , "      - (AlterSeverity \"chain.creation\" Debug)"
             , "      monitor: ((time > (23 s)) Or (time < (17 s)))"
             , "    '#aggregation.critproc.observable':"
             , "      actions:"
-            , "      - CreateMessage \"exceeded\" \"the observable has been too long too high!\""
-            , "      - AlterGlobalMinSeverity Info"
+            , "      - (CreateMessage Warning \"the observable has been too long too high!\")"
+            , "      - (SetGlobalMinimalSeverity Info)"
```

Comment:
Could parantheses be removed?

Reply:
@CodiePP what do you think?

Reply:
@365andreas Of course they can be removed. I did it just for unification with an expression. :-)

Reply:
Ok, I removed parantheses.